<launch>
  <node name="pose_node" pkg="pose" type="pose_node" respawn="false" output="screen">
     <param name="x1" value="0.5" />
     <param name="y1" value="-0.076" />
     <param name="z1" value="0.1" />
     <param name="pitch1" value="3.1415" />
     
     <param name="x2" value="0.5" />
     <param name="y2" value="-0.076" />
     <param name="z2" value="0" />
     <param name="pitch2" value="3.1415" /> 
     
     <param name="x3" value="0.3" />
     <param name="y3" value="-0.076" />
     <param name="z3" value="0.2" />
     <param name="pitch3" value="1.5709" />      
  </node>
</launch>
